src/common/exif.cc

```diff
@@ -4871,31 +4871,17 @@ void dt_transform_face_tags(Exiv2::XmpData &xmp,
     std::string regionKey =
       "Xmp.mwg-rs.Regions/mwg-rs:RegionList[" + std::to_string(i + 1) + "]";
     /* for x and y, we have to retranslate them to be the center of the region. */
-#if EXIV2_TEST_VERSION(0,28,0)
```

Comment:
Can you please comment on this? It seems quite strange to me in a PR to support better Exiv2 0.28 to remove specific code (recently added) to support 0.28... I just need to be sure there is no oversight here. TIA.